server/selinux/server/pulp-celery.te

```diff
@@ -74,7 +90,7 @@ manage_files_pattern(celery_t, pulp_var_cache_t, pulp_var_cache_t)
 
 ######################################
 
-allow celery_t var_run_t:file { write getattr read create unlink open };
+allow celery_t celery_var_run_t:file { write getattr read create unlink open };
```

Comment:
Isn't this statement redundant with L#57? I think the only one not picked up by L#57 is `unlink` but pid files don't use symlinks so it shouldn't need unlink. I think this line is safe to remove, but it would need to be retested.